great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -238,3 +239,7 @@ def add_parquet_asset(
             **kwargs,
         )
         return self.add_asset(asset)
+
+    add_csv_asset.__signature__ = _merge_signatures(add_csv_asset, CSVAsset)
```

Comment:
So will we still be expected to add to this list to get the IDE features? Does this enable auto-complete or is this just about documentation?

Reply:
Autocompletion of methods is powered by `__dir__`.
It may help with the autocompletion of keyword arguments depending on the language server the IDE is running.
But I expect there's more work todo to get this working for all IDE's (although I think it does work for jupyter).
`Help` and `inspect.Sigature` calls will show the correct signature.